src/Properties/ModelProperty.php

```diff
@@ -26,9 +26,9 @@ class ModelProperty implements PropertyReflection
     public function __construct(ClassReflection $declaringClass, Type $readableType, Type $writableType, bool $writeable = true)
     {
         $this->declaringClass = $declaringClass;
-        $this->readableType = $readableType;
-        $this->writableType = $writableType;
-        $this->writeable = $writeable;
+        $this->readableType   = $readableType;
+        $this->writableType   = $writableType;
+        $this->writeable      = $writeable;
```

Comment:
Add a new line with a variable which is longer and the whole block gets reformatted again?

(seen this in other places too in the PR)

👎 on that, IMHO unnecessary for any diff and should be disabled.

Reply:
Personally I am not fan of this rule either.